src/__tests__/BoxAnnotations-test.js

```diff
@@ -107,7 +107,7 @@ describe('BoxAnnotations', () => {
         });
 
         test('should respect default annotators if none provided', () => {
-            expect(loader.determineAnnotator(options).TYPES).toStrictEqual(['region']);
+            expect(loader.determineAnnotator(options).TYPES).toStrictEqual(['highlight', 'region']);
```

Comment:
Nit: Could we reference the enum type here?

Reply:
IIRC, we typically try to limit the coupling of test code with production code (aside from the obvious code under test)